src/ProxyManager/Generator/Util/IdentifierSuffixer.php

```diff
@@ -48,6 +48,6 @@ public static function getIdentifier(string $name): string
 
     private static function loadBaseHashSalt(): string
     {
-        return sha1(serialize(InstalledVersions::getRawData()));
+        return sha1(serialize(InstalledVersions::getAllRawData()));
```

Comment:
Can't you add the `is_callable([InstalledVersions::class, 'getAllRawData'])` check here as well? This way, people won't run into this problem like I did.

Reply:
https://github.com/Ocramius/ProxyManager/pull/700#discussion_r638675531

Reply:
yeah but it does not work